controllers/depresolver/depresolver.go

```diff
@@ -60,17 +60,19 @@ func (f LogFormat) String() string {
 }
 
 // EdgeDNSType specifies to which edge DNS is k8gb connecting
-type EdgeDNSType int
+type EdgeDNSType string
 
 const (
 	// DNSTypeNoEdgeDNS is default DNSType. Is used during integration testing when no edgeDNS provider exists
-	DNSTypeNoEdgeDNS EdgeDNSType = 1 << iota
+	DNSTypeNoEdgeDNS EdgeDNSType = "EmptyDNS"
```

Comment:
Makes sense to sync const values with const names here
```suggestion
	DNSTypeNoEdgeDNS EdgeDNSType = "NoEdgeDNS"
```